.github/scripts/apply_torch_pr.py

```diff
@@ -7,17 +7,27 @@
 
 
 parser = argparse.ArgumentParser()
-parser.add_argument('--pr-list', '-n', nargs='+', default=[], required=True)
-parser.add_argument('--repo', '-r', default='https://github.com/pytorch/pytorch.git', type=str)
+parser.add_argument('--pr-list', '-n', nargs='+',
+    default=[
+        #Upgrade submodule oneDNN to v3.4
+        "https://github.com/pytorch/pytorch/pull/122472",
```

Comment:
Can we keep previously PR apply logic and maintainer this standalone patch into `.github/script`?